README.md

```diff
@@ -305,45 +345,64 @@ The Nuxeo Platform replies with an access token:
 
 There are some utility methods on the client to help you with this flow:
 
-__Nuxeo.oauth2.getAuthorizationURL(baseURL, clientId[, params])__
+__Nuxeo.oauth2.getAuthorizationURL(opts)__
 
-Returns the OAuth2 authorization URL for the given `baseURL` and `clientId`.
+Returns the OAuth2 authorization URL for the given `opts` object, containing `baseURL`, `clientId` and optional `clientSecret` and `params`.
```

Comment:
```suggestion
Returns the OAuth2 authorization URL for the given `opts` object, containing `baseURL`, `clientId`, and optional `clientSecret` and `params`.
```

Reply:
I'm not sure, and anyway I'm not really fan of the 2 `and`s... Waiting for @ataillefer review to have a better wording 👍

Reply:
Went for:
```
Returns the OAuth2 authorization URL for the given `opts` object, containing `baseURL`, `clientId`, optional `clientSecret` and `params`.
```